sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -252,23 +275,32 @@ private File getRecordFolder() {
     /*
      * Attempts to retrieve the playback file, if it is not found an exception is thrown as playback can't continue.
      */
-    private File getRecordFile(String testName) {
-        File playbackFile = new File(getRecordFolder(), testName + ".json");
+    private File getRecordFile() {
+        File playbackFile = new File(getRecordFolder(), playbackRecordName + ".json");
+        File oldPlaybackFile = new File(getRecordFolder(), testName + ".json");
 
-        if (!playbackFile.exists()) {
+        if (!playbackFile.exists() && !oldPlaybackFile.exists()) {
             throw logger.logExceptionAsError(new RuntimeException(String.format(
-                "Missing playback file. File path: %s. ", playbackFile.getPath())));
+                "Missing both new and old playback files. Files are %s and %s.", playbackFile.getPath(),
+                oldPlaybackFile.getPath())));
         }
 
-        logger.info("==> Playback file path: " + playbackFile.getPath());
-        return playbackFile;
+        if (playbackFile.exists()) {
+            logger.info("==> Playback file path: {}", playbackFile.getPath());
+            return playbackFile;
+        } else {
+            logger.info("==> Playback file path: {}", oldPlaybackFile.getPath());
+            return oldPlaybackFile;
+        }
     }
 
-    private void writeDataToFile() throws IOException {
-        ObjectMapper mapper = new ObjectMapper();
-        mapper.enable(SerializationFeature.INDENT_OUTPUT);
-        File recordFile = createRecordFile(testName);
-        mapper.writeValue(recordFile, recordedData);
+    private void writeDataToFile() {
```

Comment:
Do we need this as a method? It's only invoked once.

Reply:
This can be inlined.